cmake/Modules/Findopm-parser.cmake

```diff
@@ -47,6 +47,9 @@ endif ()
 find_package(opm-parser CONFIG HINTS ${_opm_parser_build})
 if (opm-parser_FOUND)
     find_package(ecl REQUIRED)
+    find_package(Boost 1.44.0
+                 COMPONENTS regex filesystem system date_time
```

Comment:
`test` is private, is not exported and should be handled by each module in question.